.devcontainer/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:12-bullseye-slim
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends \
+        locales build-essential git curl sudo python3-dev python3-venv python3-pip ca-certificates unzip openssh-client \
+        default-jdk libpq-dev postgresql postgresql-client graphviz libmagic-dev ruby-dev ruby-rubygems nginx-light redis python2 \
+        libjpeg-dev libtiff-dev libwebp-dev liblcms2-dev \
+    && rm -rf /var/lib/apt/lists/* \
+    && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 \
+    && curl -O https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-5.6.16.deb \
+    && dpkg -i elasticsearch-5.6.16.deb \
+    && rm elasticsearch-5.6.16.deb \
+    && gem install compass \
+    && curl -o awscliv2.zip "https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m)-2.0.30.zip" \
```

Comment:
There's no guarantee that the output of `uname -m` will match the names used by AWS for a given architecture, I think it would be better to leave explicit as `x86_64`:
```suggestion
    && curl -o awscliv2.zip "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.0.30.zip" \
```